include/measurement_kit/http/http.hpp

```diff
@@ -98,7 +108,11 @@ void request_connect(Settings, Callback<Error, Var<net::Transport>>,
                      Var<Logger> = Logger::global());
 
 void request_send(Var<net::Transport>, Settings, Headers, std::string,
-                  Callback<Error>);
+                  Callback<Error, Var<Request>>);
```

Comment:
Ah, I didn't notice that the API changed. So, I'll crank major as part of this PR.